polyglot/piranha/src/config.rs

```diff
@@ -42,6 +42,9 @@ pub(crate) struct CommandLineArguments {
   /// Path to output summary json
   #[clap(short = 'j', long)]
   pub(crate) path_to_output_summary: Option<String>,
+  /// Should Piranha apply the replacements to the source files or not
+  #[clap(short = 'r', long)]
+  pub(crate) should_rewrite_files: Option<bool>,
```

Comment:
Is `dry_run` a better (and a more standard) phrase  ? ( I know I have been using `should_rewrite_files` in Python API ) .

Reply:
I think `dry_run` sounds better, but then the future this switch should encompass all modifications to the outside world.